src/executables/util_cs.c

```diff
@@ -4649,7 +4651,69 @@ memmon (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  /* execute phase */
+  error_code = mmon_get_server_info (server_info);
+  if (error_code != NO_ERROR)
+    {
+      goto error_exit;
+    }
+
+  // TODO: mmon_print_server_info (server_info);
+
+  if (module_index >= 0)
+    {
+      error_code = mmon_get_module_info (module_index, module_info);
+      if (error_code != NO_ERROR)
+	{
+	  goto error_exit;
+	}
+
+      // TODO: mmon_print_module_info (module_info);
+    }
+
+  if (!transaction && !module)
+    {
+      /* default case */
+      // *INDENT-OFF*
+      error_code = mmon_get_module_info_summary (std::min ((int) MMON_MODULE_LAST, 5), module_info);
```

Comment:
```suggestion
      error_code = mmon_get_module_info_summary (std::min ((int) MMON_MODULE_LAST, DEFAULT_MODULE_INFO_CNT), module_info);
```
Nit. It's better to define a constant for the kind of thing to make it self-explanatory. `constexpr int DEFAULT_MODULE_INFO_CNT=5`, this const expression even doesn't cost at all.

Reply:
Thanks. I change it.